BattleNetwork/bnEntity.cpp

```diff
@@ -1071,6 +1071,36 @@ void Entity::AdoptNextTile()
 
 void Entity::ToggleTimeFreeze(bool state)
 {
+  if (state) {
+    if (currentStatus.statusName != "None") {
```

Comment:
avoid using strings. not only are they large in size (and stored on the heap), but they are human-error prone. For instance if I type "none" on accident or say "NONe" while in a coding frenzy, we would lose our minds debugging such a simple mistake.

We already have bit flags used for HitProps. We can simply have an internal `Hit::Flag` member varible and check to see if the Freeze or Stun bits are set.